radix-transaction-scenarios/src/scenarios/account_locker.rs

```diff
@@ -3,6 +3,7 @@ use crate::utils::*;
 use radix_engine::blueprints::account::DepositEvent;
 use radix_engine::updates::*;
 use radix_engine_interface::blueprints::account::*;
+use radix_engine_interface::blueprints::locker::ResourceSpecifier;
```

Comment:
Yup. This is a change that we _used_ to need but no longer need.

Take a look at the following file:

https://github.com/radixdlt/radixdlt-scrypto/blob/feature/toolkit-receipt/radix-engine/src/transaction/mod.rs

Previously, the `toolkit_transaction_receipt` module was not public, we just re-exported all of the types from there in a `pub use`. Since there was a `ResourceSpecifier` defined there and also another `ResourceSpecifier` defined in the locker blueprint we needed to dis-ambiguate between them.

I've switched it so that the `toolkit_transaction_receipt` module is public and not re-exported which I feel is better.

Reply:
Yup, very much agreed - seems right to namespace `toolkit_transaction_receipt` as it's irrelevant to most people using the engine / scrypto stack so shouldn't be in the engine prelude 👍

Reply:
Sounds good! This has now been changed again with a new crate called `radix-engine-toolkit` that I moved the receipt models to and where the radix-engine-toolkit will eventually live once we do a mono-repo.